server/util/flagutil/flagutil.go

```diff
@@ -190,6 +194,63 @@ func (f *URLFlag) YAMLTypeAlias() reflect.Type {
 	return reflect.TypeOf((*URLFlag)(nil))
 }
 
+type FlagAlias struct {
+	name string
+}
+
+func Alias[T any](newName, name string) *T {
```

Comment:
can you double check what the "override" behavior is for normal flags before submitting this?

Reply:
Flags are last one wins, like we thought, which makes sense with how slice flags work.